tests/test_accessibility.py

```diff
@@ -312,6 +317,37 @@ def test_check_for_accessibility_errors(self):
         with self.assertRaises(AccessibilityError):
             self.page.a11y_audit.check_for_accessibility_errors()
 
+    def test_customize_ruleset_via_env_var(self):
+        with patch.dict(os.environ,
+            {'BOKCHOY_A11Y_CUSTOM_RULES_FILE': 'tests/a11y_custom_rules.js'}):
+            self.page.a11y_audit.config.customize_ruleset()
+            self._do_audit_and_check_errors(7)
+
+    def test_customize_ruleset_via_config_param(self):
```

Comment:
@benpatterson 
re: "It wasn't obvious to me, but do any of the tests cover extending the rules (e.g., both the custom rules and the axe rules are exposing errors)?"

This test and the one right above have errors resulting from both the custom ruleset and the regular ruleset. (Without the custom rule, this test would only find the 6 errors that are caught by the standard ruleset.)